src/main/java/com/star_trello/darkside/service/UserService.java

```diff
@@ -1,16 +1,50 @@
 package com.star_trello.darkside.service;
 
+import com.star_trello.darkside.dto.EditingProfileDto;
 import com.star_trello.darkside.model.User;
 import com.star_trello.darkside.repo.UserRepo;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
 
 @Service
 public class UserService {
     @Autowired
     UserRepo userRepo;
 
+    @Autowired
+    UserSessionService userSessionService;
+
     public User getUserById(int id) {
         return userRepo.getUserById(id);
     }
+
+    public ResponseEntity<?> getUserByIdWrapper(int id) {
+        User user = getUserById(id);
+        if (user == null) {
+            return ResponseEntity.notFound().build();
```

Comment:
Are it and `ResponeEntity.status(404).build()` the same things?

Reply:
synonim. make this in your manner(our project style)